bin/copyRawDataToPrm.sh

```diff
@@ -607,8 +606,9 @@ else
 	do
 		#
 		# Process this sample sheet / run.
-		#
-		filePrefix="$(basename "${sampleSheet%.*}")"
+		#	
+		fileP="$(basename "${sampleSheet}")"
+		filePrefix="$("${fileP%.*}")"
```

Comment:
This does not make sense. The original should work and the changed version should fail as ```"${fileP%.*}"``` is not a command/binary/script that can be executed in a sub-shell.

Reply:
I thought we discussed this a couple of weeks/months ago, you can't do this in one go, if your path contains dots than that will also be  included in the %.*

on the second part you are right, this doesn't make sense, that is changed now